dtrace/opte-flow-expire.d

```diff
@@ -8,20 +8,22 @@
 #include "common.h"
 #include "protos.d"
 
-#define	HDR_FMT "%-24s %-18s %s\n"
+#define	HDR_FMT "%-24s %-18s %s %s %s\n"
 
 BEGIN {
-	printf(HDR_FMT, "PORT", "FT NAME", "FLOW");
+	printf(HDR_FMT, "PORT", "FT NAME", "FLOW", "LAST_HIT", "NOW");
 	num = 0;
 }
 
 flow-expired {
 	this->port = stringof(arg0);
 	this->name = stringof(arg1);
 	this->flow = (flow_id_sdt_arg_t *)arg2;
+	this->last_hit = stringof(arg3);
+	this->now = stringof(arg4);
```

Comment:
Are these really strings, and not u64s?

Reply:
Good catch, I'll convert this to match opte-uft-hit's stucture.